moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -219,18 +219,16 @@ const ProjectView = () => {
    */
   const handleChange = useCallback(
     (event, newTab) => {
+      setSearchParams({ tab: TABS[newTab].param });
```

Comment:
setting search params does push to your browser history, so that back button works as expected.

This function does take an options object that let's you replace the history instead of add to it. E.g.

```javascript
 setSearchParams({ tab: "summary" }, { replace: true });
```

@mddilley i was thinking this might actually be something useful for the project map search params. i feel like i need to spend a full day with the react-router API to get to know all the v6 operations.

Reply:
TIL - didn't catch that option in the docs 🙏